prql/tests/integration/compile.rs

```diff
@@ -16,8 +16,8 @@ nodes:
 }
 
 #[test]
-fn transpile_variables() -> Result<()> {
-    assert_snapshot!(compile("select 1")?, @r###"
+fn transpile_variables() {
```

Comment:
Do you prefer not to have `Result`s in tests? I rather like them because it saves lots of `.unwrap`s...

Reply:
I would have to write `From<(String, Option<SourceLocation>)> for Error` for `?` to work.

That's why I removed results. There is also no ` -> Result<()>` and `Ok(())` for each function, so overall, that may be less characters of code.

Reply:
Ah OK, so just for these functions